Iplement a pixel perfect landing page

siljeloge opened the pull request:
## Description

My result so far. Added HTML and CSS, but no Javacript as i dont see what is needed. 

## Related Issue(s)

Closes #issue-number  

> Example
(#001)[https://issue-url.com/#001]

---

## Changes Made

- [ ] Feature 1 (brief description)
- [ ] Feature 2 (brief description)
- [ ] Refactoring / cleanup
- [ ] Bug fix

---

## Screenshots (if applicable)

Include screenshots or screen recordings to show UI changes or interactions.

---

## Acceptance Criteria

- [ ] Changes have been tested locally.
- [ ] Code follows the project's style guide.
- [ ] Feature aligns with the design specification.
- [ ] No console errors or warnings.
- [ ] Responsive on desktop and mobile screens.

siljeloge committed:
Submit for approval

I have only added the HTML and CSS

siljeloge committed:
Fixed image-issues

As I pushed through the last changes, the images were gone. Had to change to the right path in html.

Haimrich9 reviewed (changes requested):
The site needs to be responsive on all screen sizes. Issues that needs to be adressed:
  - Hamburger menu is missing on mobile.
  - Vertical scaling causes text to move out of place.
  - Horizontal scaling results in button overlap.
  - At screen widths below 700px, the image shrinks to less than the screen width.
  - Footer circles do not maintain their positions when scaling horizontally or vertically.
 Most of these issues are likely due to the use of % a lot. Remember to use the inspect tool to test your site on all screen sizes.
<img width="833" alt="Skjermbilde 2025-05-10 kl  20 25 51" src="https://github.com/user-attachments/assets/d9ceb66a-8714-4592-85e9-11c9954c603b" />

<img width="868" alt="Skjermbilde 2025-05-10 kl  20 26 09" src="https://github.com/user-attachments/assets/f2839ba2-5268-4951-8a23-b9e849c0bdb2" />

<img width="456" alt="Skjermbilde 2025-05-10 kl  20 27 35" src="https://github.com/user-attachments/assets/02601182-39ed-412e-b872-094b5acf6f30" />


The index.html file should be moved out of the "template" folder. It should replace the existing index.html file in the root directory, as this is the landing page.

The CSS file can be moved into the /css folder. 

<img width="114" alt="Skjermbilde 2025-05-10 kl  20 32 10" src="https://github.com/user-attachments/assets/698d0cd1-8bfe-4510-8c55-91310d9339b5" />
Feel free to reach out if there’s anything specific you need help with. I’ll be available most of the day from now until Monday

siljeloge committed:
Merge branch 'develop' into feature/landing-page

siljeloge committed:
small changes with mediaqueries

siljeloge committed:
Fixed responsive css

siljeloge committed:
added javascript and burgermenu

added a script file with javascript for the hamburgermenu. also added css and html for the hamburgermenu and mediaqueries.

Haimrich9 reviewed (changes requested):
Mobile Design starts on 768px

Footer Circles need to go below the screen like this:
<img width="320" alt="Skjermbilde 2025-05-31 kl  09 27 39" src="https://github.com/user-attachments/assets/20dbfc11-312b-4fc0-a681-49faf69baf78" />

<img width="529" alt="Skjermbilde 2025-05-31 kl  09 13 23" src="https://github.com/user-attachments/assets/271ce5a3-8b84-4815-ad79-d9411930039a" />

Buttons on mobile should go below the picture. top, should not use % as a value, since it makes the buttons move when resizing vertically
<img width="304" alt="Skjermbilde 2025-05-31 kl  09 15 23" src="https://github.com/user-attachments/assets/e56bacaa-b91e-4854-a081-8419b319fe43" />

Align hamburger menu links to the right
Make sure in the Hambuger Menu, that text and icon lines up straight
Link height should be 94px (with no padding, if padding is added, height must be decreased, so total height is 94px)

If scaling from mobile view to desktop view while the menu is open, this happens:
<img width="1236" alt="Skjermbilde 2025-05-31 kl  09 34 26" src="https://github.com/user-attachments/assets/7cae8550-c6f0-4247-b0cc-19603904eacc" />

siljeloge committed:
javascript changes

siljeloge committed:
css changes to burgermenu

siljeloge committed:
Final submission Landingpage

Added some css-code to align the links and icons on the hamburgermenu, and fixed the position of the green underline of the links on desktop-version.
